Classes/TMSDKFunctions.m

```diff
@@ -63,7 +63,9 @@ @implementation TMSDKFunctions
 NSString *TMDictionaryToQueryString(NSDictionary *dictionary) {
     NSMutableArray *parameters = [NSMutableArray array];
     
-    for (NSString *key in [[dictionary allKeys] sortedArrayUsingSelector:@selector(caseInsensitiveCompare:)]) {
```

Comment:
According to the docs [caseInsensitiveCompare:](https://developer.apple.com/documentation/foundation/nsstring/1414769-caseinsensitivecompare?language=objc) uses the comare method with option `NSCaseInsensitiveSearch` so I kept the same option.